sdk/src/test/scala/com/horizen/SidechainMemoryPoolTest.scala

```diff
@@ -0,0 +1,222 @@
+package com.horizen
+
+import java.util
+
+import org.scalatestplus.junit.JUnitSuite
+import org.junit.{Before, Test}
+import org.junit.Assert._
+import com.horizen.fixtures._
+import com.horizen.transaction.RegularTransaction
+import com.horizen.utils.FeeRate
+import org.mockito.Mockito
+import scorex.util.ModifierId
+import org.scalatestplus.mockito.MockitoSugar
+
+import scala.collection.JavaConverters._
+import scala.collection.mutable
+
+class SidechainMemoryPoolTest
```

Comment:
A few more test cases to check:
1) Create mempool with non empty map if Txs, check initial size
2) Create mempool with settings `mempool.maxSize = 0` try to add tx.